types/relay-compiler/relay-compiler-tests.ts

```diff
@@ -48,8 +48,8 @@ getLanguagePlugin(() => ({
     outputExtension: 'bar',
     typeGenerator: {
         transforms: [RelayApplyFragmentArgumentTransform.transform],
-        generate: (node, options) => {
-            visit(node, {
+        generate: (schema, node, options) => {
+            visit(schema, node, {
```

Comment:
Remember to also remove schema from here. Please refer to https://github.com/relay-tools/relay-compiler-language-typescript/pull/154/files?file-filters%5B%5D=.ts#diff-152db63141b41dd84d03a80fda984dcc line 38